platform/default/sqlite3.hpp

```diff
@@ -85,8 +87,11 @@ class Statement {
     void reset();
     void clearBindings();
 
+    int64_t lastInsertRowId() const;
+    int64_t changes() const;
```

Comment:
We can alleviate that by checking if the return value is less than 0, and returning 0 on that case - this way we can keep using unsigned 👍